### ISSUE TITLE
Is frame_missing_handler.js still relevant?

### ISSUE THREAD

**Issue by reedrolemodel** [RoleModel/rolemodel_rails#119]
[frame_missing_handler.js](../blob/master/lib/generators/rolemodel/modals/templates/app/javascript/initializers/frame_missing_handler.js) refers to a [merged issue](https://github.com/hotwired/turbo/pull/863).

**Comment by OutlawAndy** [comment]
@reedrolemodel Yes, it is.  Though that comment may be less relevant than it used to be..  Our existing Modal/Panel pattern still relies on a global `turbo:frame-missing` handler as an escape hatch after a successful form-submission redirect. 

I've experimented with a number of patterns here and weighing the various trade-offs, I still believe this is the best "general" solution.  Especially considering that this handler also serves to enable Turbo 8 morphing + scroll preservation in the case that you're redirecting back to the page which originally opened the modal/panel, as that negates a whole host of custom turbo-stream use-cases and is otherwise not possible via Turbo alone.

I'm happy to discuss alternatives on a call if you want.
